zebra/main.c

```diff
@@ -279,7 +286,14 @@ int main(int argc, char **argv)
 		case 's':
 			nl_rcvbufsize = atoi(optarg);
 			break;
+		case 'n':
 #endif /* HAVE_NETLINK */
+#if defined(HAVE_NETLINK)
+			vrf_with_netns = 1;
+#else
+			zlog_err("Netns backend requested with vrf, whereas Netlink not available. Aborting");
```

Comment:
This says we are aborting, yet we do not, we continue on.  I would expect that this should cause zebra to fail to start, since we've specified a argument that is impossible.

Reply:
ok. I will do it.